packages/playwright-test/src/runner/dispatcher.ts

```diff
@@ -512,7 +512,7 @@ export class Dispatcher {
     this._checkFinished();
   }
 
-  private _hasReachedMaxFailures() {
+  hasReachedMaxFailures(): boolean {
     const maxFailures = this._config.config.maxFailures;
     return maxFailures > 0 && this._failureCount >= maxFailures;
```

Comment:
Looking at this line, it seems straightforward to calculate `maxFailuresReached` in the reporter, instead of plumbing it. What do you think?

Reply:
we use it in various other places outside the reporter world.